Assignment2.txt

```diff
@@ -0,0 +1,26 @@
+
+1. Can two or more operators be combined in a single line of program code?
+Yes.
+
+2. What are logical errors and how are they different from syntax errors?
+The logical errors are the errors that indicate the logic in the code fails to achieve your goal, and the syntax errors are errors that exist because of the wrong using of words and grammer. The differences between both are: 1. We can receive a error message from the complishier if we made a syntax error, while we won't receive any error message if we made a logical error; 2. The logical error is the error in logic, and the syntax error is the error in grammer and words.
```

Comment:
"We can receive a error message from the complishier if we made a syntax error, while we won't receive any error message if we made a logical error;" This is true a lot of the time, but not all of the time. The main difference is that logical errors occur during runtime, whereas syntax errors occur during compilation.